src/nectarchain/dqm/mean_camera_display.py

```diff
@@ -30,10 +30,13 @@ def ConfigureForRun(self, path, Chan, Samp, Reader1):
 
     def ProcessEvent(self, evt, noped):
         pixel = evt.nectarcam.tel[0].svc.pixel_ids
-        pixel21 = np.arange(0,21,1,dtype = int)
-        pixel = list(pixel)
-        pixel21 = list(pixel21)
-        pixels = np.concatenate([pixel21,pixel])
+        if len(pixel) < self.Chan:
+            pixel21 = np.arange(0,self.Chan - len(pixel),1,dtype = int)
+            pixel = list(pixel)
+            pixel21 = list(pixel21)
+            pixels = np.concatenate([pixel21,pixel])
```

Comment:
Same comment. This may break the camera display with respect to the declared camera geometry.

Reply:
Same here: this was a misunderstanding of the logic on my side.